src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -38,6 +39,15 @@ const ApplicationDetails = ({
   const disabledApps = useSelector(
     ({ systemProfileStore }) => systemProfileStore?.disabledApps
   );
+  const centOsAppList = [
```

Comment:
We could make something that works for both RHEL and CentOS, like:

```
const osAppList = {
"centos-linux: [{ title: 'General Information', ...}],
"rhel": [{ title: 'General Information' ...}, ...],
};
```

This could be somewhere outside the component more surfaced in a constants file. 

To match them here in the component operating system name would just need to be slugified, like 

```
const osSlug = entity?.system_profile?.operating_system?.name.replace(' ', '-').toLowerCase();
```

and then `osAppList[osSlug]`.

Reply:
Originally I had some conditional logic in the const, but the issue is that this depends on the entity info, so we have to wait for that to load in. With this idea I would run into the same issue, the app would try to load in this osAppList before that osAppList meant anything.

however we cooould do this, but originally set things to an empty array until this got loaded in. Do you think that routes better? 

We cant make to components async because of react rules, so theres no await here.

Reply:
I think it is totally fine if the app list array without the OS name would be empty.

The inventory details higher up shouldn't actually attempt to render the section before it has the entity information, or? 

Even if it does now, even the header will be empty or render some skeleton loader before it has any all the info.

Reply:
It does, and the logic for this would need to be done in InventoryDetail as opposed to ApplicationDetails for it to make sense to change since thats where a lot of the logic for appList is made (a few maps and alterations). 

However, because we cant make it async, we cant rely on the entity to finish loading before the alteration functions are preformed. The osSlug will be undefined at first, and that will cause appList[osSlug] not to work. 

I can make all the alterations on the appList conditionally set afterwords, but theres a useMemo that will cause a scalprum error if call it conditionally. and if we use that useMemo with appList[osSlug], it'll break

Reply:
> because we cant make it async

What do you mean by that?

Reply:
Post gm call, I believe this new commit slaps on the alternative solution properly